runtime-devices/libimobiledevice/spec

```diff
@@ -1,5 +1,5 @@
-VER=1.3.0
-REL=2
-SRCS="tbl::https://github.com/libimobiledevice/libimobiledevice/archive/$VER.tar.gz"
-CHKSUMS="sha256::acbfb73eabee162e64c0d9de207d71c0a5f47c40cd5ad32a5097f734328ce10a"
+VER=1.3.0+git20240725
```

Comment:
Why?

Reply:
To include several fixes for Python 3.x.